src/CssToInlineStyles.php

```diff
@@ -101,7 +101,7 @@ public function getInlineStyles(\DOMElement $element)
     protected function createDomDocumentFromHtml($html)
     {
         $html = trim($html);
-        if (strstr('<?xml', $html) !== 0) {
+        if (strpos('<?xml', $html) !== 0) {
```

Comment:
unfortunately it is protected method so adding meaningful test is difficult
Initial test was added in previous PR linked in this PR